cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "http://localhost:8000",
```

Comment:
Can you access env variables at this point? If yes, this can be moved into an env variable like `HOST_URL`. If no, then we can work with this. I'm just trying to move away from hardcoding things like this as much as possible